components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx

```diff
@@ -22,15 +22,15 @@ export const MultipleChoiceGroup = (props: MultipleChoiceGroupProps): React.Reac
         key={index}
         name={field.name}
         className={className}
-        ariaDescribedBy={ariaDescribedBy}
+        {...(ariaDescribedBy && { ariaDescribedBy: ariaDescribedBy })}
```

Comment:
I can't think of a case where the Radio or Checkbox input's would need an aria-describedby. The description of the input I would think would be in it's related label and for anything lengthy and generic in the fieldset's legend and or description. But I guess if a specific checkbox really needed a lot of explaining then maybe it could be used. But I wonder if that checkbox would be packed with too much info/context.
So I left it in to be used optionally for future cases I can't think of.